### PR TITLE
fix: add the missing tooltips

### PR DESCRIPTION
**Pull request by antomor**
## What

- Add the missing tooltips

### CODE REVIEW

File: src/app/collective-rewards/rewards/builders/AllTimeShare.tsx [hunk 1]
```diff
@@ -57,7 +57,15 @@ export const AllTimeShare: FC<AllTimeShareProps> = ({ gauge, gauges, tokens: { r
   return (
     <MetricsCardWithSpinner isLoading={isLoading} borderless>
       <div className="flex flex-col gap-y-[10px]">
-        <MetricsCardTitle title="All time share" data-testid="AllTimeShare" />
+        <MetricsCardTitle
+          title="All time share"
+          data-testid="AllTimeShare"
+          tooltip={{
+            // eslint-disable-next-line quotes
+            text: "Total sum of your rewards, share from all the Builders' rewards",
```
Comment (DinizSa) [inline src/app/collective-rewards/rewards/builders/AllTimeShare.tsx:65]: Can this sentence be a bit confusing?
wdyt:
Sum of the shares your earned from all the Builders' rewards

**Comment by DinizSa** [comment]
imo triggering by just hovering (trigger="hover") the question mark icons gives a better user experience, as it decreases the number of clicks needed, wdyt?

File: src/app/collective-rewards/rewards/backers/AllTimeShare.tsx [hunk 2]
```diff
@@ -52,7 +52,15 @@ export const AllTimeShare: FC<AllTimeShareProps> = ({ gauges, tokens: { rif } })
   return (
     <MetricsCardWithSpinner isLoading={isLoading} borderless>
       <div className="flex flex-col gap-y-[10px]">
-        <MetricsCardTitle title="All time share" data-testid="AllTimeShare" />
+        <MetricsCardTitle
+          title="All time share"
+          data-testid="AllTimeShare"
+          tooltip={{
+            // eslint-disable-next-line quotes
+            text: "Total sum of your rewards, share from all the Backers' rewards",
```
Comment (DinizSa) [inline src/app/collective-rewards/rewards/backers/AllTimeShare.tsx:60]: same as [here](https://github.com/RootstockCollective/dao-frontend/pull/399/files#r1855513716)

File: src/app/collective-rewards/rewards/backers/BackerRewardsTable.tsx [hunk 3]
```diff
@@ -37,6 +37,8 @@ const tableHeaders: TableHeader[] = [
     label: 'Total Allocations',
     className: 'w-[18%]',
     sortKey: RewardsColumnKeyEnum.totalAllocationPercentage,
+    // eslint-disable-next-line quotes
+    tooltip: "Your allocations as a share % of the total Backer's allocations for the Builder",
```
Comment (DinizSa) [inline src/app/collective-rewards/rewards/backers/BackerRewardsTable.tsx:41]: suggested simplified alternative: Your percentage of the Builder total allocations

File: src/app/collective-rewards/rewards/backers/Rewards.tsx [hunk 4]
```diff
@@ -55,6 +59,10 @@ export const Rewards: FC<RewardsProps> = ({ builder, gauges, tokens }) => {
                 gauges={gauges}
                 tokens={tokens}
                 rewards={['earned', 'claimed']}
+                tooltip={{
+                  text: 'Total sum of your rewards: received and claimable',
```
Comment (DinizSa) [inline src/app/collective-rewards/rewards/backers/Rewards.tsx:63]: suggestion: Total of your received and claimable rewards

**Review by jurajpiar — APPROVED** [review]
LG

**Comment by antomor** [comment]
@DinizSa I've applied the text suggested on slack 👍🏼

**Pull request merged**